frontends/ol-components/src/components/Link/Link.tsx

```diff
@@ -5,15 +5,14 @@ import { theme } from "../ThemeProvider/ThemeProvider"
 import { LinkAdapter } from "../LinkAdapter/LinkAdapter"
 
 type LinkStyleProps = {
+  color: "black" | "white" | "red"
```

Comment:
I made `color` required because:
- it helped me check that this had no unintential changes
- in contrast to things like `nohover` and `size`, I don't think what the default  value of `color` should be is obvious.